static/src/js/components/AppLogo/AppLogo.jsx

```diff
@@ -33,8 +33,8 @@ const AppLogo = () => (
         }
         updatePath
       >
-        <span className="app-logo__site-name-ent app-logo__site-name-ent--e">Earthdata</span>
-        <span className="app-logo__site-name-ent app-logo__site-name-ent--s">Search</span>
+        <span className="app-logo__site-name-ent app-logo__site-name-ent--e">Earth</span>
+        <span className="app-logo__site-name-ent app-logo__site-name-ent--s">data Search</span>
```

Comment:
Okay, this is a minor thing but I don't think the modifier classes on these items or the html makes a ton of sense now.

I think jsx inside this link should change to

```suggestion
        <span className="app-logo__site-name-emphasis">Earth</span>
        data Search
```

And the scss would look something like this:

```scss
&__site-name {
    // ...current styles
    // the lighter font weight
}

&__site-name-emphasis {
    // the heavier font weight
}
```

Reply:
Good catch! Implemented in the latest commits